rs/nns/governance/src/proposals/update_canister_settings.rs

```diff
@@ -289,4 +285,33 @@ mod tests {
             }
         );
     }
+
+    #[cfg(feature = "test")]
+    #[test]
+    fn test_update_canister_settings_topic_mapping() {
+        use ic_base_types::CanisterId;
+        use ic_nns_constants::SNS_WASM_CANISTER_ID;
+
+        let test_cases = vec![
+            (LEDGER_CANISTER_ID, Topic::ProtocolCanisterManagement),
+            (SNS_WASM_CANISTER_ID, Topic::ServiceNervousSystemManagement),
+            (
+                CanisterId::from_u64(123_456_789),
+                Topic::NetworkCanisterManagement,
```

Comment:
What's a network canister?

Reply:
This is the current topic defined within NNS Governance, and it maps to "System Canister Management" in all the frontend interfaces. I think it's been the case since genesis, and it should be renamed later to "Application Canister Management"